module/module_test.go

```diff
@@ -1013,3 +1014,19 @@ func TestFrameSystemFromDependencies(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, th.constructCount, test.ShouldEqual, 1)
 }
+
+func TestQueryTabularDataForResource(t *testing.T) {
+	grpcClient := &inject.DataServiceClient{}
+	grpcClient.TabularDataByMQLFunc = func(ctx context.Context, in *datapb.TabularDataByMQLRequest, opts ...grpc.CallOption) (*datapb.TabularDataByMQLResponse, error) {
+		test.That(t, "my_org", test.ShouldEqual, in.OrganizationId)
+
+		return &datapb.TabularDataByMQLResponse{}, nil
+	}
+
+	os.Setenv(rutils.PrimaryOrgIDEnvVar, "my_org")
+	os.Setenv(rutils.MachinePartIDEnvVar, "part")
+
+	dataConsumer := &module.ResourceDataConsumer{}
+	dataConsumer.DataClient(context.Background(), grpcClient)
+	dataConsumer.QueryTabularDataForResource(context.Background(), "resource", nil)
```

Comment:
(minor) can we put some sort of value in this `Response` object and then `test.That` the value still exists in the response from `dataConsumer.QueryTabularDataForResource` (and that the err is nil)?